src/drive/drive-gateway.ts

```
@@ -0,0 +1,382 @@
+    }
+    
+  }catch(err){
+    return logger.Error().Url(url).Any("init", auth).Err(err).Msg("Fetch Error").ResultError();
+  }
+  
+}
+async function update(logger: Logger, fileId: string, fileMetadata: object, fileData: Blob, auth: string): Promise<object | undefined | Result<unknown>>{
+  const url = "https://www.googleapis.com/upload/drive/v3/files/";
+  let response;
+  const boundary = '-------'+generateRandom21DigitNumber();
+    var reader = new FileReader();
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)